src/challenges/reduce_but_grow.js

```diff
@@ -0,0 +1,7 @@
+function grow(x) {
```

Comment:
Try to always use variable names that reflect what is contained in the variable, like `numbers` or `nums`